typescript/src/app/newcomp/newcomp.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { NewcompComponent } from './newcomp.component';
+
+describe('NewcompComponent', () => {
+  let component: NewcompComponent;
+  let fixture: ComponentFixture<NewcompComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ NewcompComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(NewcompComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/newsbridge/sandbox/pullRequest?prid=7541583)
```suggestion
  it("should create", () => {
```